config/upload-api.yml

```diff
@@ -364,6 +364,50 @@ paths:
           schema:
             $ref: '#/definitions/Error'
 
+    get:
+      summary: Get status of validation event on a file
+      operationId: upload.lambdas.api_server.v1.area.retrieve_validation_status_and_results
+      description: |
+        Get status and results of latest validation event on a file.
+      tags:
+        - For wrangler use via API calls in scripts
+        - For ingest service use via API calls from ingest services
```

Comment:
Be careful with these tag names.  They aren't just notes, they will create separate groups of endpoints in the Swagger browser, e.g. take a look at the groups here https://upload.integration.data.humancellatlas.org/
If you want this to the grouped with the rest of the Ingest-Only endpoints, it must have an identical tag.

For example, looking at the browser right now, I would probably merge the "For Checksum Docker Image use only" and "For Validation Docker Image use only" into "Upload Service Internal Use Only".

Reply:
Thanks. Will change